api-spec/protobuf/ark/v1/service.proto

```diff
@@ -17,13 +17,24 @@ service ArkService {
       body: "*"
     };
   };
+  rpc SendTreeNonces(SendTreeNoncesRequest) returns (SendTreeNoncesResponse) {
+    option (google.api.http) = {
+      post: "/v1/payment/tree/nonces"
+      body: "*"
+    };
+  }
+  rpc SendTreeSignatures(SendTreeSignaturesRequest) returns (SendTreeSignaturesResponse) {
+    option (google.api.http) = {
+      post: "/v1/payment/tree/signatures"
+      body: "*"
+    };
+  }
```

Comment:
Some new proposal for renaming current proto has been made in https://github.com/ark-network/ark/pull/228/files

We could call these Submit..